ruma-common/src/thirdparty.rs

```diff
@@ -22,7 +22,10 @@ pub struct Protocol {
     pub location_fields: Vec<String>,
 
     /// A content URI representing an icon for the third party protocol.
+    #[cfg(not(feature = "unstable-synapse-quirks"))]
     pub icon: String,
+    #[cfg(feature = "unstable-synapse-quirks")]
+    pub icon: Option<String>,
```

Comment:
I think you'll have to duplicate the docstring, otherwise the docs will be empty with `unstable-synapse-quirks` turned on.

Reply:
Yeah, CI is failing because of this.